spec/controllers/concerns/string_redacter_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe 'StringRedacter' do
```

Comment:
looks like in the branch base switch, the `StrictRedacter` definition itself seems to have disappeared?

Reply:
Thanks for keeping an eye on this! I think this is going to have to wait until PR #7672 gets merged and I can branch directly from main.

Reply:
I also realized now, `StringRedacter` is added on that PR, but the specs, refactoring etc are all here